policy/PfsMapper.yaml

```diff
@@ -421,7 +421,7 @@ datasets:
 
   dmQaResidualStats:
     template: 'detectorMapQA/%(dateObs)s/v%(visit)06d/dmQaResidualStats-%(visit)06d-%(arm)s%(spectrograph)d.pickle'
-    python: dict
+    python: pandas.core.frame.DataFrame
     persistable: ignored
     storage: PickleStorage
```

Comment:
Not sure it's important, nor what the correct value is, but maybe we should update the `storage` value too. But if it works without it, maybe it doesn't matter.